ui/00_message/common/spot_name.xml

```diff
@@ -1928,10 +1955,13 @@
   <message>
     <key>SPOT_NAME_909</key>
     <original>ディプニエ大侵林</original>
+    <translation>Deenan Great Fores</translation>
   </message>
   <message>
     <key>SPOT_NAME_910</key>
     <original>ノーザナス侵林</original>
+    <translation>Northern Forest<translation>
+      <!--Needs revision-->
```

Comment:
If you are unsure of a name (like this one), don't add a temporary name. Add a comment, perhaps.

There is more to this name than "Northern Forest".